src/test/java/duke/EventTest.java

```diff
@@ -0,0 +1,26 @@
+package duke;
+
+import duke.tasks.Event;
+import org.junit.jupiter.api.Test;
+
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EventTest {
+    public static DateTimeFormatter VALID_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+
+    @Test
+    public void markDoneTest() {
```

Comment:
Consider naming the test methods this way?
![image](https://user-images.githubusercontent.com/66118590/91697201-48b15800-eba3-11ea-8c8c-c0067314abfc.png)